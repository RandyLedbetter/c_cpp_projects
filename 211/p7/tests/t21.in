insert Chico Sacramento 90
insert Chico Redding 73
insert Sacramento San_Francisco 87
insert Sacramento Los_Angeles 384
path Chico Los_Angeles
