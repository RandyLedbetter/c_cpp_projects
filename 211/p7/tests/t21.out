Chico --> Sacramento --> Los_Angeles
